### ISSUE TITLE
Sensors Stopped Working Possible Fix?

### ISSUE THREAD

**Issue by mattyb8562** [kevinvincent/ha-wyzesense#179]
I installed a Zwave USB stick a couple of weeks ago. When reading the documentation it included the bit below. This applied to my situation so I disabled modemanager as recommended. I was having to restart HA randomly as my sensors would stop responding, Very similar to a lot of post I have seen regarding wyze sense and HA. Since I have disabled modemanager I have not had an issue since. Could modemanager be interfering with the wyze bridge the same as it does with zwave sticks?

**"If you did an alternative install of Home Assistant on Linux (e.g., installing Ubuntu, then Docker, then Home Assistant Supervised) then the modemmanager package will interfere with any Z-Wave (or Zigbee) stick and should be removed or disabled in the host OS. Failure to do so will result in random failures of those components, e.g., dead or unreachable Z-Wave nodes"_**

**Comment by BullionMan** [comment]
When Home Assistant 115,2 dropped it made my WyzeSense in HACS setup fail. I reinstalled WyzeSense to no avail.

**Comment by jsrich31** [comment]
I am having this same issue after updating to Home Assistant 115,4/ Reinstall has not worked either

**Comment by MikeTannenbaum** [comment]
I'm still running 0.114.2 and have issues with sensors dropping and not reconnecting ¯\_(ツ)_/¯

**Comment by NotSteveAtGithub** [comment]
Quit working here in the past week, sometime around 0.115.5 to 0.115.6.  As mentioned above, reinstall did not help.

**Comment by jsrich31** [comment]
**Finally fixed this issue. There was a duplicate entry for binary_sensor: in my config file. I would run a test and everything was good and HA would work fine, but reviewing and cleaning up my files after seeing this is my logs did the trick.

YAML file /config/configuration.yaml contains duplicate key "binary_sensor". Check lines 69 and 113

All is well.....😇**

**Comment by NotSteveAtGithub** [comment]
After trying several approaches called out in some of the other issue reports (reboot with receiver unplugged, reinstall, etc.), I finally just unplugged the receiver for a few hours.  This afternoon I plugged the receiver back in, restarted HA through the UI, and now it's working again.  Didn't get anything useful from the debug output during any of my attempts to recover.  So far it seems to be magic when it quits and magic when it works again.  Here's hoping somebody figures out the root cause.

**Comment by BullionMan** [comment]
I think my two sensors lost battery juice when the system went down last
week. After reading about how the sensors seem to brick after running the
battery down completely, I am reluctant to spend even the cost of new
batteries to see if they work. I ordered an $8 Zigbee sensor to replace the
Wyze sensors, and I may never go back to the Wyze sensors. Thanks Kevin for
your excellent work on this. But I am very disappointed with Wyze in
general, as they do not have their great products available to Home
Assistant, and they generally do not make their products (other than the
cams) available to anyone residing outside of the USA.


Peter Gowdy
Toronto

**Comment by jamescadd** [comment]
@BullionMan which ZigBee sensor did you go with?  Agree that Kevin has done excellent work, but the issues extend beyond his integration.

**Comment by BullionMan** [comment]
I ordered the Sonoff Zigbee 3.0 door/window sensor off eBay, model SNZB-04
for under $9. I'm happy to be reducing the number of different
networks/radio-protocols by getting rid of the Wyze proprietary thing.
Kevin's Wyzesense solution worked very well too, but those bricking rumors.

**Comment by andystewart999** [comment]
Yeah, they aren't rumours.  I lost two Wyze Door sensors thanks to the battery going totally flat, when that happens their MAC is reset to all zeroes and that's it.

Wyze know all about it and will replace the sensors for free if you raise a ticket - but that doesn't help if you're outside of the US.  If I lose any more I'll start looking at Zigbee-based alternatives, which is a shame as the Wyze door sensors are tiny and raise changes pretty much immediately.

**Comment by BullionMan** [comment]
I am outside the US (in Canada) and I am totally down on Wyze now. They
won't ship anywhere outside the US, even Toronto, which is 30 miles across
the lake from Buffalo. I bought my WyzeCam v2 and sensor kit from Amazon
Canada. But the other Wyze products are marked up outrageously. The scale
($20 on wyze.com) is $90 Canadian or $67.50 USD + $45 (34$ USD) shipping!
And Wyze insists on using their own ecosystem within our LANs, even
bypassing local DNS, which is not cool. I happily ordered a Sonoff Zigbee
sensor for $8 and it arrived and I plopped it into my Zigbee network
immediately. I am done with Wyze for any networking stuff.


Peter Gowdy
Toronto
